ww-tech/roxie
# Update fastlane + circlci

- Update to latest version of fastlane
- Clean up Fastfile
- Explicitly set `excon` dependency to 0.71.0 due to security vulnerability. (See #16 )
- Update CircleCI config to remove `large` resource class. Jobs were not running because of this. 
    - See https://circleci.com/gh/ww-tech/roxie/19 for message about resource class not supported
